src/content/utilities/loading/loading.md

```diff
@@ -79,28 +83,21 @@ const DataComponent = ({ loading, data }) => {
 
 
 
-## Loading component
+## Inline Loading component
 
-A Loading component is a rotating shape indicating the application is processing and informs users that the system has reacted to an action taken.
+A Inline Loading component is a rotating shape indicating the application is processing and informs users that the system has reacted to an action taken. When the request action succeeds the Inline Loading Component provides feedback directly inline.
```

Comment:
- "**An** Inline Loading component" -- should be "an" not "a"
- "Component" doesn't need to be capitalized in last sentence